src/components/Facility/ConsultationDetails/QuestionnaireResponsesList.tsx

```diff
@@ -127,15 +126,22 @@ function QuestionGroup({
   }[];
   level?: number;
 }) {
+  const checkResponse = (group: Question, r_id: string) => {
```

Comment:
Should be a cleaner way to do this, something like:

<img width="559" alt="image" src="https://github.com/user-attachments/assets/eabf7cb2-5934-41c0-8656-45f2678d5485" />